src/plugins/shared_ux_services/public/services/index.tsx

```diff
@@ -39,24 +44,33 @@ const ServicesContext = createContext<SharedUXServices>(servicesFactory());
  * Within a plugin, you can use the `ServicesContext` provided by the SharedUX plugin start
  * lifeycle method.
  */
-export const ServicesProvider: FC<SharedUXServices> = ({ children, ...services }) => (
+export const SharedUXServicesProvider: FC<SharedUXServices> = ({ children, ...services }) => (
   <ServicesContext.Provider value={services}>{children}</ServicesContext.Provider>
 );
 
 /**
  * React hook for accessing the pre-wired `SharedUXServices`.
  */
-export function useServices() {
+export function useSharedUXServices() {
   return useContext(ServicesContext);
 }
 
 /**
  * React hook for accessing the pre-wired `SharedUXPlatformService`.
  */
-export const usePlatformService = () => useServices().platform;
+export const usePlatformService = () => useSharedUXServices().platform;
 
-export const usePermissions = () => useServices().permissions;
+/**
+ * React hook for accessing the pre-wired `SharedUXPermissionsService`.
+ */
+export const usePermissions = () => useSharedUXServices().permissions;
```

Comment:
do we really need a separate hook for each service? can we not just desctructure it from `useSharedUXServices`, like `useKibana`?

Reply:
I personally do like this pattern. If none of the services are related it does make sense to have a separate hook for each of them. But it comes down to dev preference 😊